server/cmwell-data-tools/src/main/scala/cmwell/tools/data/utils/logging/DataToolsLogging.scala

```diff
@@ -20,8 +20,8 @@ import org.slf4j.LoggerFactory
 case class LabelId(id: String)
 
 trait DataToolsLogging {
-  private[data] lazy val redLogger = LoggerFactory.getLogger("tools-red-logger")
-  private[data] lazy val badDataLogger = LoggerFactory.getLogger("bad-data")
+  private[data] lazy val redLogger = Logger(LoggerFactory.getLogger("tools-red-logger"))
+  private[data] lazy val badDataLogger = Logger(LoggerFactory.getLogger("bad-data"))
```

Comment:
A suggestion: maybe you should follow the "convention"  and call the logger and the log file `tools-bad-data`

Reply:
Yep - good idea. Done..